sources/Outputs/include/Dyd.h

```diff
@@ -175,8 +190,17 @@ class Dyd {
    */
   static std::vector<boost::shared_ptr<dynamicdata::MacroConnect>> writeGenConnect(const algo::GeneratorDefinition& def, unsigned int index);
 
+  /**
+   * @brief Write connections for hvdc lines
+   *   *
+   * @param collection the collection where the connection will be added
+   * @param hvdcLine the hvdc line definition to process
+   */
+  static void writeHvdcLineConnect(const boost::shared_ptr<dynamicdata::DynamicModelsCollection>& collection, const algo::HvdcLineDefinition& hvdcLine);
+
  private:
-  static const std::unordered_map<algo::GeneratorDefinition::ModelType, std::string>
+  static const std::unordered_map<algo::GeneratorDefinition::ModelType,
+                                  std::string>
```

Comment:
This should not be formatted that way. I put the whole line in once and asked for formatting and the previous version was correct

Reply:
I tried to reformat with CTRL+I or with CTRL+s and both put it in 3 differents lines, like before, did you have the comment when you tested it ?

Reply:
Put all in one line and then reformat it